packages/odyssey-design-tokens/src/transition.json

```diff
@@ -1,7 +1,8 @@
 {
   "transition": {
     "duration": {
-      "main": { "value": "100ms" }
+      "main": { "value": "100ms" },
+      "mainAsNumber": { "value": 100 }
```

Comment:
Is this what design wanted rather than a `1000ms` one?

Reply:
It's the duration for all of our transitions so, I assumed this is what we'd want. For consistency.